kafka-ballerina/records.bal

```diff
@@ -38,18 +38,18 @@ public type TopicPartition record {|
 // Security-related records
 # Configurations for secure communication with the Kafka server.
 #
-# + cert - Configurations associated with the `crypto:TrustStore`
-# + key - Configurations associated with the `crypto:KeyStore`
+# + cert - Configurations associated with the `crypto:TrustStore` or the filepath of the trustore certificate
+# + key - Configurations associated with the `crypto:KeyStore` or configuration related to `kafka:CertKey`
```

Comment:
Shall we update as follows to be consistent with other module docs?
```suggestion
# + cert - Configurations associated with crypto:TrustStore or single certificate file that the client trusts
# + key - Configurations associated with crypto:KeyStore or combination of certificate and private key of the client
```

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/240/commits/0e175487046422ca076a49b0608096d6774fedfa